From 98e966cfa6ed6a936623e74ed925a110887094cc Mon Sep 17 00:00:00 2001
From: Guillaume Ranquet <guillaumex.ranquet@intel.com>
Date: Tue, 21 May 2013 17:55:31 +0300
Subject: [PORT FROM MAIN]uevent: set permission/ownership of ttyIFX1

BZ: 94560

Fix ttyIFX1 permission/ownership for flashing through mmgr

Change-Id: I7a99d119c0041d178f1802c0ea86e60fe39ea17f
Orig-Change-Id: Ib031c2c0cbca6851cdd0c17b898d83befaa9a08e
Signed-off-by: Guillaume Ranquet <guillaumex.ranquet@intel.com>
Signed-off-by: Arun Ravindran <arun.ravindran@intel.com>
---
 rootdir/ueventd.rc |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/rootdir/ueventd.rc b/rootdir/ueventd.rc
index 0046d66..bae5af3 100644
--- a/rootdir/ueventd.rc
+++ b/rootdir/ueventd.rc
@@ -84,6 +84,7 @@
 /dev/mdmTrace             0660   root       radio
 /dev/mdm_ctrl             0660   root       radio
 /dev/ttyIFX0              0660   root       radio
+/dev/ttyIFX1              0660   root       radio
 
 # CDMA radio interface MUX
 /dev/ts0710mux*           0640   radio      radio
-- 
1.7.4.1

